plugins/jobs/src/js/data/JobModel.ts

```diff
@@ -130,6 +133,20 @@ export const resolvers = ({
       _context = {}
     ): Observable<JobLink> {
       return runJob(args.id).map(({ jobId }) => ({ jobId }));
+    },
+    createJob(
+      _parent = {},
+      args: GeneralArgs,
+      _context = {}
+    ): Observable<JobLink> {
+      return createJob(args.data).map(({ jobId }) => ({ jobId }));
```

Comment:
I got this comment on my PR from @juliangieseke. where it would be a good idea to check for the existence of the args.

I think createJob already says the type is mandatory, but since I am not sure our implementation enforces it (could be it tests query validity), you might want to protect from data not existing. 

One other idea would be to check if you can issue the graphqlObservable of a mutation without the data on args, and see if our engine complains.